universal-application-tool-0.0.1/app/views/components/Modal.java

```diff
@@ -102,6 +105,11 @@ public ModalBuilder setTriggerButtonStyles(String buttonStyles) {
       return this;
     }
 
+    public ModalBuilder setWidth(String width) {
```

Comment:
I guess the kind of funny thing about this is that it's just a style string, so really, we could just make this a general `addStyles` or `setStyles` method. Thoughts? If we made it more general, it should probably take `String... styles`

Reply:
If you don't make it more general, I think you should at least add a docstring explaining the format this method expects for the `width` argument. I.e., that it should probably be a tailwind class. Or that it will basically just be taken and applied as a class.

Reply:
I'm gonna make it an enum for modal widths. I don't want to change the styling for the general modal container too much.